static/sass/custom/_patterns_strips.scss

```diff
@@ -1,5 +1,10 @@
 @mixin jaas-p-strips {
-  $color-suru-accent: #3fc8f2;
+  @include jaas-p-strip-suru;
+  @include jaas-p-strip-slanted;
+}
+
+@mixin jaas-p-strip-suru {
+  $suru-accent: #3fc8f2;
```

Comment:
as linting suggests, where this variable is used hasn't been updated

Reply:
I dont get any suggests running `./run exec yarn run lint-scss`